Docs/Migration.md

```diff
@@ -4,6 +4,15 @@ Migrating from older versions
 ## Migrating from pre-2.8.0.0
 
 - `CustomSettingFileCheckBox` and `CustomSettingFileDropDown` have been renamed to simply `FileSettingCheckBox` and `FileSettingDropDown`. This requires adjusting the control names in `OptionsWindow.ini`. `FileSettingCheckBox` has a fallback to legacy behaviour if the control has any files defined with `FileX`.
+
 - Updater no longer has hardcoded list of download mirrors or custom components. This information must now be set in `UpdaterConfig.ini` (example is included amongst default resources in client repository). For a reference, the previously hardcoded information can be found in format used by `UpdaterConfig.ini` [here](https://gist.github.com/Starkku/1d52f0040d7a00d79e57afc2fba5f97b).
+
 - Second-stage updater no longer has hardcoded list of launcher executables to check for when restarting the client. It will now only check `ClientDefinitions.ini` for `LauncherExe` key, and it it fails to read and launch this the client will not automatically restart after updating.
-- Updater DLL filename has been changed from `DTAUpdater.dll` to `ClientUpdater.dll` and second-stage updater from `clientupdt.dat` to `SecondStageUpdater.exe` and has been moved from base folder to `Resources`.
\ No newline at end of file
+
+- Updater DLL filename has been changed from `DTAUpdater.dll` to `ClientUpdater.dll` and second-stage updater from `clientupdt.dat` to `SecondStageUpdater.exe` and has been moved from base folder to `Resources`.
+
+- To support launching the game on Linux a file named `wine-dta.sh` must be present in the game root folder. For use with wine the file should contain `wine gamemd-spawn.exe $*` where `gamemd-spawn.exe` is replaced with the game executable. Note that users will need to execute `chmod u+x wine-dta.sh` once to allow it to be launched.
```

Comment:
1. Actually the client has a special entry in clientdefinitions which allows specifying any filename, not only `wine-dta.sh`.
2. The instructions for launching the game differ for DLL injector based mods, for example, like Syringe or Vinifera.
3. Not sure about chmod'ing, wonder if the updater sets or could set the correct permissions if they are set on the FTP server...